templates/haproxy.cfg.j2

```diff
@@ -98,27 +97,27 @@ backend machine-config-server
 frontend ingress-http
     bind *:80
     default_backend ingress-http
-    mode tcp
-    option tcplog
+    mode http
+    option httplog
 
 backend ingress-http
-    balance source
-    mode tcp
+    balance roundrobin
+    mode http
```

Comment:
Would like to keep this as TCP for consistency.

Reply:
+1